security/try-pod-to-pod-encryption.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Try out the pod-to-pod encryption tech preview
+description: Try out enabling WireGuard for state-of-the-art cryptographic security between pods for Calico clusters.
+---
+
+### Big picture
+
+Enable WireGuard to secure pod-to-pod traffic in a {{site.prodname}} cluster.
+
+> **Warning!** Pod-to-pod encryption is a tech preview and should not be used in production clusters. The functionality is not supported with overlay networks (in particular, NodePort) and using NodePort from within a non host-networked will not work. It has had very limited testing and it will contain bugs (please report these on the Calico Users slack or GitHub). In addition, it does not support all the features of {{site.prodname}}.
+{: .alert .alert-danger }
+
+### Value
+
+{{ site.prodname }} supports WireGuard tunnels between nodes providing transport-level security for pod-to-pod traffic. WireGuard provides [formally verified](https://www.wireguard.com/formal-verification/) secure and [performant tunnels](https://www.wireguard.com/performance/) without any specialized hardware. For a deep dive in to WireGuard implementation, see [whitepaper](https://www.wireguard.com/papers/wireguard.pdf).
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- [Felix resource]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition) with WireGuard configuration parameters.
```

Comment:
Delete the link...should be

- **Felix resource** with WireGuard configuration parameters

Reply:
Wait, shouldn't this be "FelixConfiguration" resource? we don't have a "Felix resource"

@realgaurav @bmckercher123

Reply:
Using **Felix configuration resource**